tests/kafkatest/tests/streams/base_streams_test.py

```diff
@@ -98,5 +98,9 @@ def wait_for_verification(self, processor, message, file, num_lines=1):
     @staticmethod
     def verify_from_file(processor, message, file):
         result = processor.node.account.ssh_output("grep -E '%s' %s | wc -l" % (message, file), allow_fail=False)
-        return int(result)
+        try:
+          return int(result)
+        except ValueError:
+          self.logger.warn("Command failed with " + result)
+          return False
```

Comment:
Sorry, `False` is not a number ;)

Seriously, it looks like the usage of this method expects a number, so if we're coercing a failure, we should explicitly coerce it to a number.

Reply:
🤦‍♀

Reply:
I agree, apparently without strong typing I just lose all sense of type whatsoever